examples/iot/index.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
+
+import * as azure from "@pulumi/azure";
+import * as iot from "@pulumi/azure/iot";
+
+const location = "West US 2";
+
+const resourceGroup = new azure.core.ResourceGroup("test", {
+    location: location,
+});
+
+const iotHub = new iot.IoTHub("test", {
+    resourceGroupName: resourceGroup.name,
+    sku: {
+        capacity: 1,
+        name: "S1",
+        tier: "Standard",
+    }
+});
+
+iotHub.onEvent("test", async (context, arg) => {
+    console.log("ctx: " + JSON.stringify(context, null, 4));
+    console.log("arg: " + JSON.stringify(arg, null, 4));
```

Comment:
`arg` is just a string, so we can log it as-is without `JSON.stringify`. I know you probably copied it from the other sample, that one can be simplified too,

Reply:
Fixed in 4befc8cc16a7a663e608029b69f0fd4a8d66b1e2